float64 depth
float64 yaw_rate
float64 forwards_velocity
float64 sideways_velocity
